runtime/altair/Cargo.toml

```diff
@@ -260,18 +264,23 @@ std = [
   "pallet-uniques/std",
   "pallet-utility/std",
   "pallet-vesting/std",
+  "pallet-whitelist/std",
   "pallet-xcm/std",
   "pallet-xcm-transactor/std",
   "parachain-info/std",
+  "parity-scale-codec/std",
+  "getrandom/std",
```

Comment:
Where is this crate used?

Reply:
Great find! Turns out we have used this crate throughout all runtimes without needing it. Moreover, my commit duplicated a few `std` derivations for some crates.

* Cleanup of multiple deps in Altair: [9a4950a](https://github.com/centrifuge/centrifuge-chain/pull/1828/commits/9a4950a5095dac845053a980fa6484de68717bae)
* Removal of `getrandom` for all runtimes: [bc2ae52](https://github.com/centrifuge/centrifuge-chain/pull/1828/commits/bc2ae5273ac07eb93c38806fe0db0294601815d9)

Reply:
Thanks!!